package.json

```diff
@@ -138,6 +138,9 @@
     "@storybook/addon-notes@5.3.21": {
       "react-syntax-highlighter": ">=15.5.0"
     },
+    "@storybook/storybook-deployer@^2.8.11": {
```

Comment:
The `^` doesn't make that much sense to me. The specific version or not having the version is better IMHO.

Reply:
Ok, I will remove that

Reply:
Done!